.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -29,7 +29,7 @@ body:
 
   - type: input
     attributes:
-      label: vizro version
+      label: vizro or vizro-ai version
```

Comment:
Wondering if we should just have a new template for `vizro-ai` or is there a way how we can add a `vizro `or vizro-ai label automatically to the issue report?

Either way I am fine though

Reply:
Hm we could maybe add something so that they choose which package they are referring to

Reply:
That would be nice - an additional question maybe? Is there a way how we can attach the answer to that question to a label being automatically added? 

If this takes a long time to figure out, then just ignore and we go forward with adding it manually for now

Reply:
https://stackoverflow.com/questions/69571617/github-issue-form-add-label-based-on-dropdown

At least from this it looks like it wasn't figured out